### PR TITLE
Switch CI project stackrox-ci -> acs-san-stackroxci

### PR DESCRIPTION
**Pull request by Stringy**
## Description

The current stackrox-ci GCP project is redundant and will soon be removed. This PR moves our test VMs over to a new GCP project (acs-san-stackroxci) and reconfigures SSH keys, VM networking, and service accounts, to allow us to use this new project.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Updated documentation accordingly

**Automated testing**
  - [ ] Added unit tests
  - [ ] Added integration tests
  - [ ] Added regression tests

If any of these don't apply, please comment below.

## Testing Performed

Tested VM creation and interaction locally (in dev setup, using acs-sandbox project) and relying on CI to test new service account use, and VM creation and interaction

### CODE REVIEW

File: .github/workflows/integration-tests-vm-type.yml [hunk 1]
```diff
@@ -107,7 +107,9 @@ jobs:
 
       - name: Teardown VMs
         if: always()
-        run: make -C "${{ github.workspace }}/ansible" destroy-vms
+        run: |
+          make -C "${{ github.workspace }}/ansible" list-inventory
```
Comment (Molter73) [inline .github/workflows/integration-tests-vm-type.yml:111]: Is this line for debugging? Do we intend to keep it?

**Review by Molter73 — COMMENTED** [review]
LGTM! Just left a small comment.

Went through the CI errors, what I could find:
- Fedora fails due to the newer kernel not being supported by us yet.
- RHEL seems to have a flake pulling the collector image.
- Ubuntu has a single test failing that might be a flake.

File: .github/workflows/integration-tests-vm-type.yml (hunk 1 above)
Comment (Stringy) [reply .github/workflows/integration-tests-vm-type.yml:111]: ahh yeah it was for debugging. I don't think it's strictly necessary, though hiding it behind GHA debug flag might be pretty useful:

```
        if [[ "${RUNNER_DEBUG}" == "1" ]]; then
          make -C "${{ github.workspace }}/ansible" list-inventory
        fi
```

WDYT?

**Review by Molter73 — APPROVED** [review]
LGTM!

**Pull request merged**